client-common/src/tendermint/rpc_client/sync_rpc_client.rs

```diff
@@ -116,7 +116,12 @@ impl SyncRpcClient {
         heights: T,
     ) -> Result<Vec<ValidatorsResponse>> {
         let params = heights
-            .map(|height| ("validators", vec![json!(height.to_string())]))
+            .map(|height| {
+                (
+                    "validators",
+                    vec![json!(height.to_string()), json!("0"), json!("100")],
```

Comment:
what's that 0 and 100? paging?

Reply:
yes, the rpc call changed, if not add 0, 100, there will be error.

Reply:
`page` and `per_page`? I think we shouldn't have an assumptions of 100 validators?

Reply:
https://docs.tendermint.com/master/rpc/#/Info/validators

Reply:
default is 30, the max is 100, I don't know what value should be used here, I used the max value.